src/routes/docs/products/auth/teams/+page.markdoc

```diff
@@ -215,4 +215,19 @@ individual roles in the team using the `Role.team(<TEAM_ID>, [<ROLE_1>, <ROLE_2>
 
 {% arrow_link href="/docs/advanced/platform/permissions" %}
 Learn more about permissions
-{% /arrow_link %}
\ No newline at end of file
+{% /arrow_link %}
+
+# Memberships privacy {% #memberships-privacy %}
+
+Hide specific membership details from other users by marking them as private. To set this up, go to **Auth** > **Security** > **Memberships privacy**
+
+These details can be made private:
+
+- `userName` - The member's name
+- `userEmail` - The member's email address
+- `mfa` - Whether the member has enabled multi-factor authentication
+
+{% info title="Scale plan feature" %}
+Memberships privacy is available on the Appwrite Cloud scale plan.
+{% /info %}
```

Comment:
```suggestion
```

We don't need this anymore since we're going to make this feature available to all plans.